src/utils/tests/helpers.ts

```diff
@@ -181,12 +181,14 @@ export const testRepoState: DockerRepoState = {
 export const getNetwork = (
   networkId = 1,
   name?: string,
+  description?: string,
```

Comment:
I would suggest moving this to be the last parameter of this method so that you do not have to update all of the calls to this method in the many tests.

Another approach is to just remove this parameter here and only set the description in the tests that need to update it. Ex:
```
const network = getNetwork(1, 'test network', Status.Started, 2);
network.description = 'my description';
```